modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -1676,6 +1676,9 @@ label:default-output[] label:new[Introduced in 5.9]
 == DROP CONSTRAINT
```

Comment:
We might want to also update the createStatement result column description from show to reflect that it can be null for vector type properties:
```
| createStatement
| Statement used to create the constraint, or `null` if the constraint cannot be created through `CREATE CONSTRAINT` (for example in case of version incompatibilities).
| STRING
```

Reply:
Good point! Please feel free to push the change to this PR.